src/test/java/easytests/api/v1/controllers/SubjectsControllerTest.java

```diff
@@ -133,9 +133,85 @@ public void testListForbidden() throws Exception {
     /**
      * create
      */
-    /**
-     * update
-     */
+    @Test
+    public void testUpdateSuccess() throws Exception {
+        final SubjectModelInterface subjectModel = subjectsSupport.getModelFixtureMock(0);
+
+        when(this.subjectsService.find(any(), any())).thenReturn(subjectModel);
+        when(this.acl.hasAccess(any(UserModelInterface.class))).thenReturn(true);
+
+        mvc.perform(put("/v1/subjects")
+                .contentType(MediaType.APPLICATION_JSON)
+                .content(new JsonSupport()
+                        .with(id, 1)
```

Comment:
really you need to take that fields from additionalModelMock(1)

Reply:
done